defaults/main.yml

```diff
@@ -103,7 +103,7 @@ cis_sshd_client_alive_interval: 300
 cis_sshd_client_alive_count_max: 0
 cis_sshd_login_grace_time: 60
 cis_sshd_allow_users: "ec2-user vagrant" # A comma-delimited list of users to allow ssh access to.
-cis_sshd_allow_groups: "ec2-user" # A comma-delimited list of groups to allow ssh access to
+cis_sshd_allow_groups: "ec2-user vagrant" # A comma-delimited list of groups to allow ssh access to
```

Comment:
I don't think this should be a default.
we should just have `ec2-user` as a default and it can be overridden when testing with vagrant.

Reply:
the vagrant user won't exist in EKS case, that's why I kept it. Let me see if I can conditionally exclude it, I'd like to keep vagrant testing route functional.

Reply:
addressed in 9f873f3a88b5d3fbfc23ed5080aa89bd5a3d5c45